### PR TITLE
Slim down version with ES6

### PR DESCRIPTION
**Pull request by davidjamesherzog**
Create slimmed down version of ad markers video.js plugin with ES6

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
```
Comment (pwalton) [inline .editorconfig:1]: probably don't need this file in here

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+dist: xenial
```
Comment (pwalton) [inline .travis.yml:1]: We don't use Travis anymore

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# CONTRIBUTING
```
Comment (pwalton) [inline CONTRIBUTING.md:1]: Are you planning on making this public?

File: .editorconfig (hunk 1 above)
Comment (davidjamesherzog) [reply .editorconfig:1]: we do use this in our IDE

**Pull request merged**